packages/main/test/pages/TimePicker.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta charset="utf-8">
+
+		<script>
+			delete Document.prototype.adoptedStyleSheets;
+		</script>
+
+		<script src="../../webcomponentsjs/webcomponents-loader.js"></script>
+		<script src="../../resources/bundle.esm.js" type="module"></script>
+		<script nomodule src="../../resources/bundle.es5.js"></script>
+
+		<script
+		src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+		integrity="sha256-3edrmyuQ0w65f8gfBsqowzjJe2iM6n0nKciPUp8y+7E="></script>
```

Comment:
side thing: please add html, body {height: 100%}  so the page does not look odd (if you inspect the body tag it appear small)

Reply:
Done